prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -4,8 +4,19 @@ import com.google.common.net.HttpHeaders
 import com.microsoft.azure.functions.HttpRequestMessage
 import com.microsoft.azure.functions.HttpResponseMessage
 import com.microsoft.azure.functions.HttpStatus
+import gov.cdc.prime.router.PAYLOAD_MAX_BYTES
+import java.io.File
+import java.io.IOException
+import java.net.HttpURLConnection
+import java.net.URL
 import java.time.OffsetDateTime
 
+enum class ReportStreamEnv(val endPoint: String) {
+    TEST("https://pdhtest-functionapp.azurewebsites.net/api/reports"),
+    LOCAL("http://localhost:7071/api/reports"),
+    STAGING("https://pdhstaging-functionapp.azurewebsites.net/api/reports")
+}
```

Comment:
I created this little enum in support of the new TestReportStream, but then put it here since it might be generally useful.